apps/system/js/app_chrome.js

```diff
@@ -855,6 +858,9 @@
       this.setThemeColor('');
     }
     this.setSiteIcon();
+    if (this.pinned) {
+      this.pin();
```

Comment:
Same as above, this is causing the browser chrome to expand and then collapse again on navigation. Let's leave this to bug 1168962.